### PR TITLE
[documentation] Add section for systemd service and nginx to installation guide

### PR DESCRIPTION
**Pull request by zoe-bat**

### CODE REVIEW

File: docs/installation_guide/binary.md [hunk 1]
```diff
@@ -46,14 +46,15 @@ cp ./example/config.yaml .
 
 Now open the file in your text editor of choice so that you can set some important configuration values. Change the following settings:
 
-* Set `host` to whatever hostname you're going to be running the server on (eg., `example.org`).
-* Set `port` to `443`.
-* Set `db-type` to `sqlite`.
-* Set `db-address` to `sqlite.db`.
-* Set `storage-local-base-path` to the storage directory you created above (eg., `/gotosocial/storage`).
-* Set `letsencrypt-cert-dir` to the certificate storage directory you created above (eg., `/gotosocial/storage/certs`).
+- Set `host` to whatever hostname you're going to be running the server on (eg., `example.org`).
```
Comment (tsmethurst) [inline docs/installation_guide/binary.md:49]: The changes in this file should be undone -- the dot stuff is the old way of doing config, which will be changed in the next release.

**Comment by tsmethurst** [comment]
Mostly looks good, just make sure to add a nav entry in the appropriate place here: https://github.com/zoe-bat/gotosocial/blob/docs-update/mkdocs.yml#L14

**Comment by zoe-bat** [comment]
Thank you! Must have had an old version of the binary guide.
The things should be fixed now!

File: docs/installation_guide/nginx.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+## Configure GoToSocial
+
+In your GoToSocial config turn off letsencrypt.
+First open the file in your text editor.
+
+```bash
+sudoedit /gotosocial/config.yaml
+```
+
+Then set `letsencrypt-enabled: true`.
```
Comment (tsmethurst) [inline docs/installation_guide/nginx.md:35]: this should be false right? :P

Comment (zoe-bat) [reply docs/installation_guide/nginx.md]: it should i am going to scream

**Pull request merged**